couchapps/bib_input/evently/bib_output_last_contestants/_changes/opts.json

```diff
@@ -0,0 +1,4 @@
+{
+  "id":"checkpoints",
+  "filter": "bib_input/checkpoints"
```

Comment:
needs the corresponding
$.evently.changesDBs[app.db.name]["checkpoints"].opts.site_id = app.site_id;

Reply:
fixed